modules/apps/site-initializer/site-initializer-testray/site-initializer-testray/extra/remote-app/src/pages/Project/Suites/SuiteOutlet.tsx

```diff
@@ -64,7 +64,7 @@ const SuiteOutlet = () => {
 	});
 
 	useEffect(() => {
-		setHeaderActions({actions, item: testraySuite, mutate});
+		setHeaderActions({actions: [], item: testraySuite, mutate});
```

Comment:
It would be better that you make this validation in a form component.
```suggestion
		setHeaderActions({actions, item: testraySuite, mutate});
```